### PR TITLE
Add github actions basic CI

### PR DESCRIPTION
**Pull request by d-asensio**
## Description

Add CI to avoid merging broken builds

### CODE REVIEW

**Comment by d-asensio** [comment]
@XaviTorello here we go! I added this to avoid what happened in #26

**Review by XaviTorello — CHANGES_REQUESTED** [review]
Great @d-asensio :) 

Just a proposal to also take care of our forks triggering also on new PR

File: .github/workflows/push-check.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+name: Push Check Pipeline
+
+on: [push]
```
[the remark this answers is not in the document]
Comment (d-asensio) [reply .github/workflows/push-check.yml:4]: You're right, working...🤖

[the remark this answers is not in the document]
Comment (d-asensio) [reply .github/workflows/push-check.yml]: Done!

**Review by XaviTorello — APPROVED** [review]
Perfect!

**Pull request merged**